litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/Panel/PanelContent.tsx

```diff
@@ -0,0 +1,314 @@
+                </IconButton>
+              </Tooltip>
+            ) : (
+              <Tooltip
+                title={`${t('analyticsDashboard.toolTip.viewChaosMetric')}`}
+              >
+                <IconButton
+                  className={classes.panelIconButton}
+                  onClick={() => {
+                    setViewEventMetric(true);
+                  }}
+                >
+                  <ViewChaosMetric className={classes.panelIcon} />
+                </IconButton>
+              </Tooltip>
+            )}
+            <Tooltip title={`${t('analyticsDashboard.toolTip.editPanel')}`}>
+              <IconButton
+                disabled
+                className={classes.panelIconButton}
+                onClick={() => {}}
+              >
+                <Edit className={classes.panelIcon} />
+              </IconButton>
+            </Tooltip>
+            <Tooltip title={`${t('analyticsDashboard.toolTip.popout')}`}>
+              <IconButton
+                className={classes.panelIconButton}
+                onClick={() => {
+                  setPopout(true);
+                }}
+              >
+                <Expand className={classes.panelIcon} />
+              </IconButton>
+            </Tooltip>
+          </div>
+        </div>
+        <div>
+          <Modal
+            open={popout}
+            onClose={() => setPopout(false)}
+            disableBackdropClick
+            disableEscapeKeyDown
+            modalActions={
+              <ButtonOutlined onClick={() => setPopout(false)}>
+                &#x2715;
+              </ButtonOutlined>
+            }
+            height="95% !important"
+            width="95%"
+          >
+            <div className={classes.popOutModal}>
+              <Typography className={classes.title}>{panel_name}</Typography>
+              <LineAreaGraph
+                legendTableHeight={120}
+                openSeries={graphData}
+                eventSeries={chaos_data}
+                showPoints={false}
+                showLegendTable
+                showEventTable
+                showTips
+                showEventMarkers
+                marginLeftEventTable={10}
+                unit={unit}
+                yLabel={y_axis_left}
+                yLabelOffset={55}
+                margin={{ left: 75, right: 20, top: 20, bottom: 10 }}
+              />
+            </div>
+          </Modal>
+        </div>
+        <div className={classes.singleGraph}>
+          <LineAreaGraph
+            legendTableHeight={120}
+            openSeries={graphData}
+            eventSeries={chaos_data}
+            showPoints={false}
+            showEventTable={viewEventMetric}
+            showLegendTable
+            showTips
+            showEventMarkers
+            unit={unit}
+            yLabel={y_axis_left}
+            yLabelOffset={55}
+            margin={{ left: 75, right: 20, top: 20, bottom: 10 }}
```

Comment:
can we change the LineAreaGraph API to take a classname for overall outer stylings like margins and padding @whitetiger1399 
you can look into it if you like...we have done so for other components of litmus-ui